daml-lf/archive/BUILD.bazel

```diff
@@ -51,6 +51,7 @@ load("@rules_pkg//:pkg.bzl", "pkg_tar")
         ),
     ]
     for version in LF_VERSIONS
+    if version != "1.12"
```

Comment:
Why don’t we produce protobufs for preview versions? Did we do the same for 1.11?

Reply:
Or is this because the ledger API side (including daml-lf) is unchanged? I think we should still have those libs. Omitting certain versions just gets confusing.

Reply:
I did not want to release the proto for 1.12.  I will.